From: Bastian Blank <waldi@debian.org>
Date: Sun, 11 Feb 2018 06:24:15 +0000
Subject: liblockdep: Stub NMI watchdog reset
Forwarded: https://lore.kernel.org/lkml/20180828233721.GF18030@decadent.org.uk/T/#u

lockdep.c now includes <linux/nmi.h> and requires touch_nmi_watchdog(),
so provide those for liblockdep.

Fixes: 88f1c87de11a ("locking/lockdep: Avoid triggering hardlockup from ...")
[bwh: Write a longer description]
Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 tools/include/linux/nmi.h | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 tools/include/linux/nmi.h

--- /dev/null
+++ b/tools/include/linux/nmi.h
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/*
+ *  linux/include/linux/nmi.h
+ */
+#ifndef LINUX_NMI_H
+#define LINUX_NMI_H
+
+static inline void touch_nmi_watchdog(void)
+{
+}
+
+#endif
